crates/core/tedge/src/main.rs

```diff
@@ -19,6 +19,7 @@ const BROKER_GROUP: &str = "mosquitto";
 
 fn main() -> anyhow::Result<()> {
     let opt = cli::Opt::parse();
+    tedge_utils::logging::initialise_tracing_subscriber(false);
```

Comment:
Is not it should be like below, this also needs to pass --debug flag when one uses the tedge cli.
```suggestion
    tedge_utils::logging::initialise_tracing_subscriber(opt.debug);
```

Reply:
This might be updated when merged with https://github.com/thin-edge/thin-edge.io/pull/1624.

Reply:
For this, we need to add `--debug` option with `tedge`  command. This can be a separate PR.